<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberWindow</class>
 <widget class="QMainWindow" name="MemberWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidgetusers">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QPushButton" name="pushButtonsearch">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>90</y>
          <width>161</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>查看菜系</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtoncomment">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>230</y>
          <width>161</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>提交意见</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonover">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>370</y>
          <width>161</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="DishWidget" name="widgetmenu_2" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>761</width>
          <height>411</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonusersback">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>480</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>返回上一级</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>60</y>
          <width>351</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>华文琥珀</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 0);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="radioButtonAnonymous">
           <property name="text">
            <string>匿名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonRealname">
           <property name="text">
            <string>实名</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>160</y>
          <width>491</width>
          <height>271</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
font: 14pt &quot;楷体&quot;;
background-color: transparent;
border-color: rgb(0, 0, 0);
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>5</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonrelease">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>460</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>发布意见</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonclear">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>460</y>
          <width>111</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>清空文本并返回</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>100</y>
          <width>301</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>请在此编辑框内编辑</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DishWidget</class>
   <extends>QWidget</extends>
   <header location="global">dishwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
